### DISCUSSION TITLE
Boot a self-configured ISO image via the PXE server

### DISCUSSION THREAD

**Discussion by ASIPDNJIOQWND** [ipxe/ipxe#1038] [Q&A]
I prepared an ISO image that sets up the installation automatically, but that did not work because after installing the PXE server I get the window where I have to choose whether to try or install Ubuntu and this window should not appear after the change. and I tried this via the USB stick and that worked .

**Comment by NiKiZe** [comment]
If you get messages in Ubuntu you need to turn to them.
If anyone should be able to help you you need to give much more info. All your script and configuration files as a start.

**Comment by ASIPDNJIOQWND** [comment]
Hi NiKiZe,

here is the page (https://linuxhint.com/pxe_boot_ubuntu_server/) I took for the PXE setup and I took all configuration from there.

Best
Alex

**Reply by NiKiZe** [reply]
To summarize that page it builds iPXE (with several invalid targets) creates NFS root, extracts ISO, uses embedded script and tftp transfer (http is recommended) 

The guide shows the window in Ubuntu UI so your description matches the guide.

If you have a USB that does automation, what changes was made to the USB to have it automated? And have you tried to replicate those changes into the file structure used for PXE booting?
